index.js

```diff
@@ -3,6 +3,6 @@
 var scan = require('./scan')
 var parse = require('./parse')
 
-module.exports = function (source) {
-  return parse(scan(source))
+module.exports = function (source, validateLicenseNames = true) {
```

Comment:
typically i'd expect absence and false to be the same for boolean options; this should probably be inverted so it can default to false?

Reply:
You are absolutely right, I will change this.